src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -44,11 +62,11 @@ def _timeout_to_deadline(self, timeout: int) -> Optional[int]:
     def __call__(self,
                  request,
```

Comment:
What should be the type for request? I would suggest using `typing.Generic` for the class, and use `TypeVar` here.